tests/e2e/specs/table.cy.js

```diff
@@ -182,18 +180,17 @@ describe('State saving', () => {
       .get('.c-table table > tbody > tr')
       .should('have.length', initialNumRows)
     cy
-      .get('[data-cy="filter task state"]:last')
+      .get('[data-cy=control-taskStateFilter]:last')
       .click()
     cy
-      .get('.v-list-item')
-      .contains(TaskState.SUCCEEDED.name)
-      .click({ force: true })
+      .get('.v-list-item[state=succeeded]')
+      .click({ force: true, multiple: true })
```

Comment:
This suggests the menu for the tree view tab is also being rendered (although probably has `display: none`). Normally vuetify does not render a menu until the first time it is opened (which was the case before this PR).